server/controllers/notes.js

```diff
@@ -1,30 +1,59 @@
-const Notes = require('../models/notes')
+const Note = require('../models/notes');
 
 const notesController = {
     create: (req, res) => {
-        // code here
-        console.log(`create: ${req.body}`)
-        res.send('ok')
+        const note = new Note({
+            title: req.body.title,
+            notes: req.body.notes
+        });
```

Comment:
Let's follow one practice throughout. If we have semicolons everywhere, keep these. Of we don't remove these. :)